PlayDate/Views/Cells/MatchTableViewCell.swift

```diff
@@ -8,16 +8,89 @@
 import UIKit
 
 class MatchTableViewCell: UITableViewCell {
+    
+    static let identifier = "MatchTableViewCell"
+    
+//    static let appUser: User = {
+//        let user = User(userId: <#T##Int#>, name: <#T##String#>, bio: <#T##String#>, birthday: <#T##String#>, age: <#T##Int#>, pets: <#T##[Pet]#>)
+//        return user
+//    }()
```

Comment:
Do we need this in the future? If not, can we remove them just to clean up the code a little?